integration/spark/src/test/scala/org/apache/carbondata/spark/testsuite/cleanfiles/TestCleanFilesCommandPartitionTable.scala

```diff
@@ -0,0 +1,361 @@
+    checkAnswer(sql(s"""select count(*) from si_cleantest"""), Seq(Row(4)))
+
+    assert(FileFactory.isFileExist(mainTableTrashFolderPath))
+
+    count = 0
+    var listMainTable = getFileCountInTrashFolder(mainTableTrashFolderPath)
+    assert(listMainTable == 8)
+
+    // recovering data from trash folder
+    val timeStamp = getTimestampFolderName(mainTableTrashFolderPath)
+
+    val segment0Path = mainTableTrashFolderPath + CarbonCommonConstants.FILE_SEPARATOR +
+      timeStamp + CarbonCommonConstants.FILE_SEPARATOR + CarbonCommonConstants.LOAD_FOLDER + '0'
+    val segment1Path = mainTableTrashFolderPath + CarbonCommonConstants.FILE_SEPARATOR +
+      timeStamp + CarbonCommonConstants.FILE_SEPARATOR + CarbonCommonConstants.LOAD_FOLDER + '1'
+    val segment2Path = mainTableTrashFolderPath + CarbonCommonConstants.FILE_SEPARATOR +
+      timeStamp + CarbonCommonConstants.FILE_SEPARATOR + CarbonCommonConstants.LOAD_FOLDER + '2'
+    val segment3Path = mainTableTrashFolderPath + CarbonCommonConstants.FILE_SEPARATOR +
+      timeStamp + CarbonCommonConstants.FILE_SEPARATOR + CarbonCommonConstants.LOAD_FOLDER + '3'
+
+    sql(s"CREATE TABLE c1 USING CARBON LOCATION '$segment0Path'")
+    sql("INSERT INTO cleantest select * from c1").show()
+    sql("drop table c1")
+
+    sql(s"CREATE TABLE c1 USING CARBON LOCATION '$segment1Path'")
+    sql("INSERT INTO cleantest select * from c1").show()
+    sql("drop table c1")
+
+    sql(s"CREATE TABLE c1 USING CARBON LOCATION '$segment2Path'")
+    sql("INSERT INTO cleantest select * from c1").show()
+    sql("drop table c1")
+
+    sql(s"CREATE TABLE c1 USING CARBON LOCATION '$segment3Path'")
+    sql("INSERT INTO cleantest select * from c1").show()
+    sql("drop table c1")
+
+    checkAnswer(sql(s"""select count(*) from cleantest"""),
+      Seq(Row(4)))
+    sql(s"CLEAN FILES FOR TABLE cleantest options('force'='true')").show
+    // no files in trash anymore
+    count = 0
+    listMainTable = getFileCountInTrashFolder(mainTableTrashFolderPath)
+    assert(listMainTable == 0)
+    sql("show segments for table cleantest").show()
+    sql("show segments for table si_cleantest").show()
+    sql("""DROP TABLE IF EXISTS CLEANTEST""")
+  }
+
+  def editTableStatusFile(carbonTablePath: String) : Unit = {
+    val f1 = new File(carbonTablePath + CarbonCommonConstants.FILE_SEPARATOR + "Metadata" +
+      CarbonCommonConstants.FILE_SEPARATOR + "tablestatus")  // Original File
+    val f2 = new File(carbonTablePath + CarbonCommonConstants.FILE_SEPARATOR + "Metadata" +
+      CarbonCommonConstants.FILE_SEPARATOR + "tmp") // Temporary File
+    val w = new PrintWriter(f2)
+    Source.fromFile(f1).getLines
+      .map { x =>
+        x.replaceAll("Success", "In Progress")
+      }
+      // scalastyle:off println
+      .foreach(x => w.println(x))
+    // scalastyle:on println
+    w.close()
+    f2.renameTo(f1)
+  }
+
+
+  def getFileCountInTrashFolder(dirPath: String) : Int = {
```

Comment:
reformat the method

Reply:
i don't understand, what do i have to do here?

Reply:
same as above